protocols/872-cosmosid-ngs-library-prep-part1/nexteraXT_dna_library_prep_part1.ot2.py

```diff
@@ -0,0 +1,104 @@
+from opentrons import labware, instruments, robot
+
+"""
+Tagment Genomic DNA
+Amplify Libraries
+"""
+# labware setup
+gDNA_plate = labware.load('96-PCR-flat', '1')
+out_plate = labware.load('96-PCR-flat', '2')
+# out_plate = labware.load('biorad-hardshell-96-PCR', '2')
+tuberack = labware.load('tube-rack-2ml', '5')
+
+# reagent setup
+atm = tuberack.wells('A1')  # Amplicon Tagment Mix
+td = tuberack.wells('B1')  # Tagment DNA Buffer
+nt = tuberack.wells('C1')  # Neutralize Tagment Buffer
+npm = tuberack.wells('D1')  # Nextera PCR Master Mix
+index_1 = tuberack.wells('A2', to='D4')  # Index 1 (i7) adapters
```

Comment:
This is a comment for when we put on master branch (don't need to change now). "Index_1" and "Index_2" should be "Index_7" and "Index_5"